mmf_cli/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
What is the advantage of adding these symlinks? Why not move the files entirely here?